lib/routes/account.js

```diff
@@ -972,6 +979,7 @@ module.exports = function (
       },
       handler: function (request, reply) {
         log.begin('Account.devices', request)
+        var language = request.query.language || 'en'
```

Comment:
I don't mind using the accept-language header. I Think @shane-tomlinson mentioned that it would be better to use the language returned from `/config` of the content server. Thoughts?